### DISCUSSION TITLE
My School has multiple E-Mail endpoints, please help.

### DISCUSSION THREAD

**Discussion by Logxn** [community/community#61620] [GitHub Education]
### Hi everyone,

Hello guys,

I used to pass the application process for GitHub education just fine.
That is until GitHub forced the users of my school to verify their enrollment via the schools e-mail.

My School however has two separate endpoints for e-mail because of different security needs.
Only one endpoint is currently recognized @ GitHub.

Now I also allowed my location that proves that I am near campus, I uploaded a document proving I am enrolled the only part that is missing is my endpoint.
How can I get GitHub to recognize a second school email address.

Example:
student@foo.de is recognized
student@bar.de is not recognized but belong to the same school, just two different course of studies.

Thank you!

![image](https://github.com/community/community/assets/20424274/3163f8f7-babd-40ba-bc59-2de5d0b134a9)

### A suggestion...(optional)

Allow multiple E-Mail endpoints.

**Comment by samus-aran** [comment]
Hi @Logxn (again) 👋 

We've located your support ticket(s) and will respond when we have any updates regarding the school info update request 👍

**Reply by Logxn** [reply]
Hello @samus-aran 

GitHub finally managed to associate my e-mail endpoint with my university.
However now I am not allowed to use the best proof method because it tells me I have already used those proof methods in the past. (????)
And secondly if I use a new type of proof I cant because I supposedly lack my full name and bio on my profile, which is simply not true.
